tasklist/client/src/Tasks/CollapsiblePanel/CustomFiltersModal/DeleteFilterModal.tsx

```diff
@@ -53,7 +52,19 @@ const DeleteFilterModal: React.FC<Props> = ({
 
         onDelete();
       }}
-    />
+    >
+      <p>
```

Comment:
Without the `<p>` tag, the text would still be wrapped in the `<div>` and styled according to the class applied to the`<div>`. However, it would lose its semantic meaning as a paragraph, which could impact accessibility.